joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/primefaces/PrimefacesAutoConfiguration.java

```diff
@@ -88,4 +94,11 @@
 	@EnableConfigurationProperties(Primefaces6_2Properties.class)
 	public static class Primefaces6_2AutoConfiguration {
 	}
+	
+	@Bean("primefaces")
+	@ConditionalOnMissingBean(name = "primefaces")
+	@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE, proxyMode = ScopedProxyMode.TARGET_CLASS)
```

Comment:
`PrimeFaces.getInstance()` seems to return the very same object instance on every invocation, regardless of the current request or session context. Therefore `prototype` scope seems unnecessary.

Reply:
@larsgrefer I followed the exact pattern of JsfAutoConfiguraton which returns FacesContext.getCurrentInstance() the same exact way as a prototype and not proxied.

Reply:
```java
	@Bean("facesContext")
	@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE, proxyMode = ScopedProxyMode.TARGET_CLASS)
	@ConditionalOnMissingBean
	public FacesContext facesContext() {
		return FacesContext.getCurrentInstance();
	}
```

Reply:
But `FacesContext.getCurrentInstance()` returns different object instances depending on the context (thread) its called in. Therefore the corresponding spring bean uses the prototype scope, so every time it's used `FacesContext.getCurrentInstance()` is invoked to obtain the correct `FacesContext` instance (and not the instance from 5 requests ago).

Reply:
Understood. I just removed the Scope annotation.